full-service/src/service/wallet_service.rs

```diff
@@ -101,9 +103,68 @@ impl<
             logger,
         }
     }
+}
+
+impl<
+    T: BlockchainConnection + UserTxConnection + 'static,
+    FPR: FogPubkeyResolver + Send + Sync + 'static,
+> Wallet for WalletService<T, FPR> {
```

Comment:
I don't think any of this code is new. It just moved around because these functions are now declared in the Wallet trait.